src/discovery/local-discovery.js

```diff
@@ -194,7 +194,7 @@ export class LocalDiscovery extends TypedEmitter {
     const remoteId = keyToPublicId(remotePublicKey)
 
     this.#log(
-      `${isInitiator ? 'outgoing' : 'incoming'} secretSteam connection ${
+      `${isInitiator ? 'outgoing' : 'incoming'} secretStream connection ${
```

Comment:
nit: `secret stream` or `SecretStream`

Reply:
Changed in 29df1d0301d8b646b3fa04c061061ad6e8ee7db6.